tests/run.sh

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env bash
 
 IMG=${1}
-DIR=${2}
+DASH=${2}
 
 running_in_docker() {
   awk -F/ '$2 == "docker"' /proc/self/cgroup 2> /dev/null | read || return
 }
 
 if running_in_docker; then
```

Comment:
Tested the docker incantation described in the README and also executed tests/run.sh via the local temp directory method: `bash tests/run.sh reichlab reichlab/flusight-dashboard` (after building the docker and tagging the image as reichlab)

👍